recipes/android-ndk/all/conanfile.py

```diff
@@ -281,8 +281,13 @@ def package_info(self):
         # when `tools.android:ndk_path` is provided, so it MUST NOT be manually injected here to `tools.cmake.cmaketoolchain:user_toolchain` conf_info
         self.conf_info.define("tools.android:ndk_path", os.path.join(self.package_folder, "bin"))
 
-        self.buildenv_info.define_path("CC", self._define_tool_var("CC", "clang"))
-        self.buildenv_info.define_path("CXX", self._define_tool_var("CXX", "clang++"))
+        compiler_executables = {
+            "c": self._define_tool_var("CC", "clang"),
+            "cpp": self._define_tool_var("CXX", "clang++"),
+        }
```

Comment:
@franramirez688 should it be replaced by:

```suggestion
        compiler_executables = self.conf.get("tools.build:compiler_executables", default={}, check_type=str)
        compiler_executables["c"] = self._define_tool_var("CC", "clang")
        compiler_executables["cpp"] = self._define_tool_var("CXX", "clang++")
```

Reply:
Good point! 👏 

Anyway, I recall that we could simplify these lines:

```python
            compiler_executables = {
	            "c": self._define_tool_var("CC", "clang"),
	            "cpp": self._define_tool_var("CXX", "clang++"),
	        }
            self.conf_info.update("tools.build:compiler_executables", compiler_executables)
```
If I'm not wrong, this should do the trick.